frontend-react/src/views/Add Drive/index.js

```diff
@@ -0,0 +1,208 @@
+
+    return result;
+  });
+
+  const classes = useStyles();
+  
+  return (
+    <Page title="Drive" className={classes.root}>
+      <Container className={classes.container} maxWidth="md">
+        {Object.keys(formik.errors).length > 0 ? (
+          <Box className={classes.errorContainer}>
+            <Typography
+              variant="h4"
+              component="h4"
+              className={classes.errorTitle}
+            >
+              {errorTitle}
+            </Typography>
+            <ul>
+              {Object.values(formik.errors).map((errorText) => (
+                <li key={errorText}>{errorText}</li>
+              ))}
+            </ul>
+          </Box>
+        ) : null}
+        <form className={classes.formContainer} onSubmit={formik.handleSubmit}>
+          <Typography variant="h2" component="h2" className={classes.title}>
+            {title}
+          </Typography>
+          <Box display="flex" flexDirection="column">
+            {fieldList.map((field) => {
+              if (!field.isTextField) {
+                const { translatedLabel, labelName, items, errorText } = field;
+                return (
+                  <WhiteBox key={`${labelName}-select-container`}>
+                    <FormControl variant="outlined" fullWidth>
+                      <InputLabel id={`drive-${labelName}-label`}>
+                        {translatedLabel}
+                      </InputLabel>
+                      <Select
+                        id={`drive-${labelName}`}
+                        name={labelName}
+                        labelId={`drive-${labelName}-label`}
+                        label={translatedLabel}
+                        value={formik.values[labelName]}
+                        onChange={formik.handleChange}
+                        error={!!errorText}
+                      >
+                        {items.map((item) => (
+                          <MenuItem key={item} value={item}>
+                            {item}
+                          </MenuItem>
+                        ))}
+                      </Select>
+                    </FormControl>
+                  </WhiteBox>
+                );
+              } else {
+                const { translatedLabel, labelName, type, errorText } = field;
+                return (
+                  <WhiteBox key={`${labelName}-container`}>
+                    <TextField
+                      id={`drive-${labelName}`}
+                      name={labelName}
+                      variant="outlined"
+                      fullWidth
+                      label={translatedLabel}
+                      type={type}
+                      value={formik.values[labelName]}
+                      onChange={formik.handleChange}
+                      error={!!errorText}
+                    />
+                  </WhiteBox>
+                );
+              }
+            })}
+          </Box>
+          <p>
+            {traveled} km {useT('traveled')}
```

Comment:
```suggestion
            {traveled} km {traveledT}
```
please keep the `useT` out of returned JSX template, it's easier to find them when they're at the beginning of the function rather than somewhere in the returned template.